### PR TITLE
Task 2-3

### PR DESCRIPTION
**Pull request by Gamedar**

### CODE REVIEW

**Comment by Gamedar** [comment]
@lexastef please review.

File: .idea/bsu-internship.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lexastef) [inline .idea/bsu-internship.iml:1]: .idea files should be in a gitignore

Comment (lexastef) [reply .idea/bsu-internship.iml:1]: Use `git rm -r .idea` t exclude them from index.

File: UI/css/styles.css [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+}
+
+a:hover {
+    text-decoration: underline;
+    cursor: pointer;
+}
+
+input, button {
+    border: none;
+    padding: 12px !important;
```
Comment (lexastef) [inline UI/css/styles.css:21]: Why important?

Comment (Gamedar) [reply UI/css/styles.css:21]: The search input field has so padding without it

File: .idea/bsu-internship.iml (hunk 1 above)
Comment (Gamedar) [reply .idea/bsu-internship.iml]: Fixed

**Comment by Gamedar** [comment]
1. Post hashtags are implemented, I don't see what's the issue there.
2. Why a separate design section for filtering is needed? My design has a search field in the upper left and all filtering can be done on backend through tokenizing a search query and analyzing it for a presence of hashtags, dates or nicknames.

Twitter doesn't have any filtering design element. 
image

3. I will implement pagination in future commits.

File: UI/css/styles.css (hunk 2 above)
Comment (lexastef) [reply UI/css/styles.css:21]: You need to fix this in direct places or do not override your pro[erties.
Browser defaults has the minor specificity.
Important for "global" styles unacceptable.

**Comment by lexastef** [comment]
Filter was described in technical specification so should implement it in one variation or another
In case you want to have clever logic for filtering in one search field (like twitter) - that's ok, but
TO ACCEPT lab2 it should be described in details or implemented (nobody done such thing on the practice from my experience :) )

Mockups is for lab 2 is not just "something", it careful and detailed work.

**Pull request merged**